site/content/docs/5.2/examples/form/index.html

```diff
@@ -56,7 +56,7 @@ <h1 class="title">Travel</h1>
           <li class="nav-item">
             <a href="#" class="nav-link nav-icon">
               <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
-                <use xlink:href="#user"/>
+                <use xlink:href="#user"></use>
```

Comment:
Should we change all the `<use.*/>` inside the code then ? We have many of them inside cards example but also in our doc.

Reply:
I haven't changed anything in terms of consistency because it's not consistent neither in Bootstrap.